docs/resources/widgets/widgets.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Overview
+---
+# 
+## What's a widget?
```

Comment:
I think we should have a page in the left hand side bar for this page (right now, I have to click on the widget section to get to this page) to be consistent with the projects section. It wasn't really clear to me this existed from the left hand side panel. Perhaps we call it "What's a widget?"